rosetta-lang/src/main/java/com/regnosys/rosetta/generator/java/object/ModelObjectBuilderGenerator.xtend

```diff
@@ -173,6 +174,7 @@ class ModelObjectBuilderGenerator {
 		'''
 		«IF attribute.isMulti»
 			@Override
+			@«RosettaAttribute»("«attribute.javaAnnotation»")
```

Comment:
Moved `@RosettaAttribute` from `setListAttribute` to `addListAttribute`, which is the convention in Jackson.

Reply:
Alternatively, I could modify the default Jackson behaviour in rosetta-common?

Reply:
See diffs in `PojoRegressionTest`